datadog_lambda/asm.py

```diff
@@ -223,7 +223,9 @@ def get_asm_blocked_response(
         content = ""
     else:
         content_type = blocked.get("content-type", "application/json")
-        content = http_utils._get_blocked_template(content_type)
+        content = http_utils._get_blocked_template(
+            content_type, blocked.get("block_id", "default")
+        )
```

Comment:
This really should have a test. Something to confirm that we're passing the correct values to the `_get_blocked_template` method.